src/pages/TermsOfService.js

```diff
@@ -83,12 +86,12 @@ const TermsOfServicePage = () => {
           }
         }, [tosText])
       ]),
-      !noActionNeeded && !usageAllowed && !!tosText &&
+      !acceptedLatestTos && !usageAllowed && !!tosText &&
```

Comment:
Nit: do we need to check `!acceptedLatestTos` here? Doesn't `!usageAllowed` imply `!acceptedLatestTos`?